cognite/client/_api/simulators/simulator_models.py

```diff
@@ -179,3 +187,136 @@ def retrieve(self, id: int | None = None, external_id: str | None = None) -> Sim
             identifiers=identifiers,
             resource_path="/simulators/models",
         )
+
+    @overload
+    def create(self, model: Sequence[SimulatorModel]) -> SimulatorModelList: ...
+
+    @overload
+    def create(self, model: SimulatorModel | SimulatorModelWrite) -> SimulatorModelList: ...
+
+    def create(
+        self, models: SimulatorModel | SimulatorModelWrite | Sequence[SimulatorModel] | Sequence[SimulatorModelWrite]
+    ) -> SimulatorModel | SimulatorModelList:
+        """`Create simulator models <https://developer.cognite.com/api#tag/Simulator-Models/operation/create_simulator_model_simulators_models_post>`_
+
+        You can create an arbitrary number of simulator models, and the SDK will split the request into multiple requests.
+
+        Args:
+            models (SimulatorModel | SimulatorModelWrite | Sequence[SimulatorModel] | Sequence[SimulatorModelWrite]): No description.
+
+        Returns:
+            SimulatorModel | SimulatorModelList: Created simulator model(s)
+
+        Examples:
+
+            Create new simulator models::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import SimulatorModelWrite
+                >>> client = CogniteClient()
+                >>> models = [SimulatorModelWrite(name="model1"), SimulatorModelWrite(name="model2")]
+                >>> res = client.simulators.models.create(models)
+
+        """
+        assert_type(models, "simulator_model", [SimulatorModelCore, Sequence])
+
+        return self._create_multiple(
+            list_cls=SimulatorModelList,
+            resource_cls=SimulatorModel,
+            items=models,
+            input_resource_cls=SimulatorModelWrite,
+            resource_path="/simulators/models",
+        )
```

Comment:
for this operation to work you have to set `limit=1` otherwise this request will fail when multiple models are passed in.

Reply:
Set `self._CREATE_LIMIT = 1` in `__init__` and this will be taken care of automatically.

Reply:
The base `resource_path` here should be set as a class variable, I believe `_RESOURCE_PATH`. Then you can use:
```py
resource_path=self._RESOURCE_PATH,
# ...and similarity for other endpoints, e.g.:
resource_path=self._RESOURCE_PATH + "/revisions",
```